jina/drivers/__init__.py

```diff
@@ -266,6 +269,54 @@ def __call__(self, *args, **kwargs):
         self._traverse_apply(self.req.docs, *args, **kwargs)
 
     def _traverse_apply(self, docs: Iterable['jina_pb2.Document'], *args, **kwargs) -> None:
+        if os.getenv('JINA_USE_TREE_TRAVERSAL', 'False') == 'True':
```

Comment:
Note for the future, if we have some toggle like this that can affect performance, I would put it as argument so that it can be used on docker containers and Remote Peas.

Reply:
Ah yes, you can even do that in the `__init__` function and "hang in" the right function:
```
if os.getenv("JINA_USE_TREE_TRAVERSAL", "False"):
    self._traverse_apply = self._traverse_apply_explicit
else:
    self._traverse_apply = self._traverse_apply_recur
```